applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing_improved_integration.h

```diff
@@ -90,14 +90,18 @@ class MapperVertexMorphingImprovedIntegration : public MapperVertexMorphing
     // --------------------------------------------------------------------------
     void Initialize() override
     {
-        if (mIsMappingInitialized == false)
-        {
-            SetIntegrationMethod();
-            FindNeighbourConditions();
-        }
+        SetIntegrationMethod();
+        FindNeighbourConditions();
 
         MapperVertexMorphing::Initialize();
     }
+
+    void Update() override
```

Comment:
```suggestion
    void UpdateInterface() override
```
then it would be consistent with the MappingApp
FYI In the near future I am planning to move the baseclass to the core so that you can also use it

Reply:
In the long run, using a common mapping structure makes totally sense. But I am not sure, whether Interface is the proper expression in this context. Your call, though.

Reply:
I would also prefer not to rename in this MR, since its not related.